packages/decent.js/docs/channel.md

```diff
@@ -0,0 +1,60 @@
+# const channel = client.channels.nth(0)
+Represents a single channel. Updated automatically.
+
+---
+
+Properties:
+
+* channel.id (string)
+* channel.name (string)
+* channel.deleted (boolean)
```

Comment:
When would this be used? Does the channel not just disappear when deleted?

Reply:
If you have already have a Channel and it gets deleted, it is removed from `client.channels` but your reference can't just become null; `channel.deleted = true` is done instead.

```js
const channel = client.channels.find(c => c.name === 'general')

await channel.delete()

channel.deleted //=> true
client.channels.find(c => c.name === 'general') //=> undefined
```

Reply:
Ah. I suppose that makes sense.